src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -214,7 +214,7 @@ public Set<Modifier> modifiers() {
 
         /**
          * {@return an unmodifiable set of the module {@linkplain AccessFlag
-         * requires flags, possibly empty}}
+         * requires flags}, possibly empty}
```

Comment:
The link fix looks okay but the wording in the new accessFlags() method is different to the wording in the other methods. The other methods use "possibly-empty unmodifiable" in the return description. Either is fine but I think we should try to keep the wording consistent where possible.

Reply:
I've looked over the wording of the various accessFlags methods. The wording for ModuleDescriptor.Requires looks consistent to me:

Returns an unmodifiable set of the module flags, possibly empty.

Returns an unmodifiable set of the module export flags for this module descriptor, possibly empty.

Returns an unmodifiable set of the module opens flags, possibly empty.

Returns an unmodifiable set of the module requires flags, possibly empty.

----

Returns an unmodifiable set of the access flags for this class, possibly empty.

Returns an unmodifiable set of the access flags for the executable represented by this object, possibly empty.

Returns an unmodifiable set of the access flags for this member, possibly empty.

Returns an unmodifiable set of the access flags for the parameter represented by this object, possibly empty.

Am I overlooking something in the twisty passages, all alike?

Reply:
If you compare the javadoc for the modifiers vs. accessFlags in each of ModuleDescriptor, Requires, Exports, and Opens then you'll see what I mean. I was hoping they could be inconsistent.

Reply:
> If you compare the javadoc for the modifiers vs. accessFlags in each of ModuleDescriptor, Requires, Exports, and Opens then you'll see what I mean. I was hoping they could be consistent.

Ah, okay. Pushed docs updated as suggested for consistency within ModuleDescriptor.java.